create_rental.rb

```diff
@@ -0,0 +1,21 @@
+require './rental'
+
+class CreateRental
+  def initialize(rent)
+    @rentals = rent
+  end
+
+  def create_rental(app)
+    print "Select a book from the following list by number \n"
+    app.list_books
+    book = gets.chomp.to_i
+    print "select a person \n"
+    app.list_people
+    person = gets.chomp.to_i
+    print 'select a date'
+    date = gets.chomp
+    new_rental = Rental.new(date, app.books[book], app.people[person])
+    @rentals << new_rental
+    puts 'Rental created successfully'
```

Comment:
- [ ] Please it would be better if you separate all the requests for input and input validation into private methods this way you achieve single responsibility principle